cdap-api/src/main/java/co/cask/cdap/api/schedule/Schedule.java

```diff
@@ -34,12 +38,14 @@ public Schedule(String name, String description, String cronEntry) {
     this.name = name;
     this.description = description;
     this.cronEntry = cronEntry;
+    this.runRequirements = null;
```

Comment:
this constructor is marked ``@Deprecated`` but doesn't say what should be used and when we should remove this usage, etc.